SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -295,47 +292,29 @@ void USpatialNetDriver::InitiateConnectionToSpatialOS(const FURL& URL)
 	Connection->OnConnectedCallback.BindUObject(this, &USpatialNetDriver::OnConnectionToSpatialOSSucceeded);
 	Connection->OnFailedToConnectCallback.BindUObject(this, &USpatialNetDriver::OnConnectionToSpatialOSFailed);
 
-	bool bUseReceptionist = true;
-	bool bShouldLoadFromURL = true;
-
 	// If this is the first connection try using the command line arguments to setup the config objects.
 	// If arguments can not be found we will use the regular flow of loading from the input URL.
 	if (!GameInstance->GetFirstConnectionToSpatialOSAttempted())
 	{
-		bool bSuccessfullyLoadedFromCommandLine;
-		StartSetupConnectionConfigFromCommandLine(bSuccessfullyLoadedFromCommandLine, bUseReceptionist);
-		bShouldLoadFromURL = !bSuccessfullyLoadedFromCommandLine;
 		GameInstance->SetFirstConnectionToSpatialOSAttempted();
+		if (!TrySetupConnectionConfigFromCommandLine())
+		{
+			SetupConnectionConfigFromURL(URL);
+		}
 	}
 	else if (URL.Host == SpatialConstants::RECONNECT_USING_COMMANDLINE_ARGUMENTS)
 	{
-		bool bSuccessfullyLoadedFromCommandLine;
-		StartSetupConnectionConfigFromCommandLine(bSuccessfullyLoadedFromCommandLine, bUseReceptionist);
-
-		if (!bSuccessfullyLoadedFromCommandLine)
+		if (TrySetupConnectionConfigFromCommandLine())
```

Comment:
Oh this should be !TrySetupConnectionConfigFromCommandLine()? If we can't load from the command line then we should use the default receptionist maybe.

Reply:
oh yeah, good catch. I was playing around with that and then forgot to set it back 🤦‍♀